### ISSUE TITLE
Integrate Accessibility Testing into CircleCI Pipeline

### ISSUE THREAD

**Issue by DanielJDufour** [GSA/code-gov-front-end#127]
**Is your feature request related to a problem? Please describe.**

We should be able to see if any PR impacts accessibility.

**Describe the solution you'd like**

We should find a package that we can run that identifies accessibility errors in our project and throws an error if there are any.  We can then add this into the CircleCI testing and integration pipeline, so that we see a CircleCI workflow step doesn't pass when there are accessibility problems. 

**Describe alternatives you've considered**

Are there any websites that provide this is a service / Github plugin like CodeClimate and LGTM.com? @froi or @saracope , do you know of any?

**Additional context**

**Comment by saracope** [comment]
I have used [pa11y](http://pa11y.org/), which has a cli tool.

**Comment by froi** [comment]
We can use the pa11y cli in our circleci scripts. Have it be part of the build step with the linter.

**Comment by saracope** [comment]
I've also been checking out [Google Lighthouse](https://developers.google.com/web/tools/lighthouse/) recently. I like how it gives a scored report for the metrics.

**Comment by saracope** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Implement tests on front-end](https://trello.com/c/Oq8Qqv5A/1196-implement-tests-on-front-end)

**Comment by DanielJDufour** [comment]
Anyone working on this? I'm happy to work on this.

**Comment by saracope** [comment]
I haven't added it here yet. I started adding it to the styleguide before going on vacay figuring that would have less issues to fix because it's a smaller site. I'll try to find time to finish that today and maybe you can look at it for me?

**Comment by saracope** [comment]
https://github.com/GSA/code-gov-style/pull/54

**Comment by DanielJDufour** [comment]
Awesome!
